### ISSUE TITLE
SVG foreignObject rendering

### ISSUE THREAD

**Issue by ssured** [knsv/mermaid#180]
For me the delivered SVG does not render in a separate window, but when adding xmlns for the HTML elements in a `foreignObject` it does work.

This snippet does not render TEXT
```XML
...
<foreignObject width="131" height="18">
  <div style="display: inline-block; white-space: nowrap;">
    TEXT
  </div>
</foreignObject>
...
```

This snippet does render TEXT
```XML
...
<foreignObject width="131" height="18">
  <div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">
    TEXT
  </div>
</foreignObject>
...
```

**Comment by knsv** [comment]
Hi sorry for the late response!

Is this still an issue?
If so in what browser do you run this?

**Comment by ssured** [comment]
I guess Chrome, it is a while ago. You should provide a xmlns when blending various xml based languages

**Comment by knsv** [comment]
Sounds like a good idea. Will add it.

Thanks for the input.

**Comment by knsv** [comment]
Fix for this issue was released in mermaid 0.5.2. Will close this now, let me know if you issues still remain.
